AMaDda/Screens/ProfileModal/Controllers/ProfileModalViewController.swift

```diff
@@ -6,24 +6,48 @@
 //
 
 import UIKit
-import SwiftUI
+
+// TODO: edit, add viewController에서 delegate 구현 필요
+protocol profileModalViewDelegate: AnyObject {
+    func registerSelectedCharacter(imageName: String)
+}
 
 final class ProfileModalViewController: UIViewController {
-    
     // MARK: - Properties
-    enum Size {
+    private enum Size {
         static let horizontalInset: CGFloat = 20.0
-        static let collectionInterItemSize: CGFloat = 5.0
+        static let collectionInterItemSize: CGFloat = 20.0
     }
+
+    // TODO: 이미지 이름 바인딩 필요
+    var selectedCharacterName = "Character1"
+    private var postfixNums = Array<Int>(1...8)
+    private lazy var characterNames: [String] = {
+        let names = postfixNums.map {
+            "Character" + String($0)
+        }
+        return names
+    }()
```

Comment:
```suggestion
    private lazy var characterNames: [String] = postfixNums.map { "Character" + String($0) }
```

이렇게는 안될까요 .. ???!!!! (컴파일될지 안될지 노자신.. )

Reply:
+ Young의 조언
enum을 사용하면 오타도 방지하고, 더욱 효율적으로 사용할 수 있어요 ~

Reply:
enum을 효율적으로 사용하는 방법에 대해 알아가고 있는 것 같습니다 ~ 감사합니다 ㅎㅎ
반영 완료하였습니다 d2bdce2f428cb1b4a9cfee62512c9db335831569